src/java.base/share/classes/java/util/Random.java

```diff
@@ -149,6 +285,18 @@ public Random(long seed) {
     private static long initialScramble(long seed) {
         return (seed ^ multiplier) & mask;
     }
+    /**
+     * Returns an instance of {@link Random} based on this
+     * {@code java.util.random.RandomGenerator}. If this generator is already an instance of
```

Comment:
Would it make sense to use a `{@link ...}` here instead?
(and maybe remove the `{@link ...}` from the `@param` and the `@return`; since that might be rather uncommon for OpenJDK javadoc?)